spec/std/system_spec.cr

```diff
@@ -18,4 +18,12 @@ describe System do
       cpu_count.should eq(shell_cpus)
     end
   end
+
+  describe "login" do
+    it "returns String when found" do
+      shell_login = `echo $USER`.strip
```

Comment:
@RX14 I was using your recommendation from https://github.com/crystal-lang/crystal/pull/4800#discussion_r131544877 .
I'll change to using ENV.

As to the utility, if I log in using `user1` and later do `sudo su - user2`, System.login will return `user1`, whereas `ENV["USER"]` will return `user2`.

For process-specific owner lookup, an enhancement to `Process` implementing the `getuid` call would do what you're asking about.